azure-pipelines.yml

```diff
@@ -17,6 +17,7 @@ stages:
   - template: azure-templates/stages.yml
     parameters:
       stageName: 'BuildComplexCase'
+      disableDiff: True
 
 # Test Versions - At least 1 of each supported LabVIEW version and 1 of each bitness
       lvVersionsToBuild:
```

Comment:
option: could add a third optional property to the lvVersionsToBuild e.g. 
 - version: '2023'
   bitness: '64bit'
   diff: true

This would require the yaml files to be edited on every custom device, but would give wider control over what version of labview is used for diffs (e.g. maybe in the future a version of LabVIEW is much faster and we want that one, or maybe something like seecd wants 2 diffs since it builds on multiple bitnesses, etc.).  Maybe worth?

Reply:
The diff logic in the python code relies on 64-bit versions only (although it should work with 32-bit with more variables being exposed). However, Scan Engine is the only repo that uses 32-bit LV for builds, and the number of files it impacts are minimal. Deferring for now.